client/src/app/shared/shared-share-modal/video-share.component.html

```diff
@@ -173,6 +173,13 @@ <h4 i18n class="modal-title">Share</h4>
                 i18n-labelText labelText="Loop"
               ></my-peertube-checkbox>
             </div>
+
+            <div *ngIf="!isLocalVideo() && !isVideoInEmbedTab()" class="form-group">
+              <my-peertube-checkbox
+                inputName="originUrl" [(ngModel)]="customizations.originUrl"
+                i18n-labelText labelText="Origin URL"
```

Comment:
Maybe `Use origin instance URL` instead?

Reply:
Good point, I changed it.